libvips/include/vips/vips.h

```diff
@@ -158,6 +158,8 @@ extern "C" {
 				  -1) \
 			: vips_init(ARGV0))
 
+VIPS_API
+int vips_max_coord_get(void);
```

Comment:
I wondered about that, but `MAX_COORD` is used to make int ranges for params, so changing after types have been built won't have any effect. It can only be done right at the start of `VIPS_INIT`.

Reply:
Ah, I see. Makes sense.

Reply:
I wanted to leave it as a function, since user code can have `VIPS_MAX_COORD` and having an int variable across library boundaries is usually a bad idea.